tox.ini

```diff
@@ -56,4 +56,4 @@ extras =
     pyside6: pyside6
 commands_pre =
     pyqt6,pyside6: pip install -U pytest-qt@git+https://github.com/pytest-dev/pytest-qt.git
-commands = pytest --color=yes --cov=superqt --cov-report=xml {posargs}
+commands = pytest --color=yes --cov=superqt --cov-report=xml --pyargs superqt {posargs}
```

Comment:
thanks for this.  I'm assuming this is to fix errors like https://github.com/napari/superqt/runs/3203181005

can you help me understand why it fixes that?

Reply:
The `pyargs` told `pytest` that tested part of code is installed python package. 
Earlier problem was connected with two locations of of same test files. They suggest using this in the documentation:
https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-as-part-of-application-code

They also suggest using `src` layout to avoid problems with the identification of which part of the code is tested: 

https://docs.pytest.org/en/6.2.x/goodpractices.html#tests-outside-application-code